python/sunbeam/schedule.py

```diff
@@ -16,6 +16,22 @@ def wells(self):
     def group(self, name):
         return Group(self._group(name), self)
 
+    def group_tree(self, timestep):
+        return GroupTree(self._group_tree(timestep))
+
+    def gtree(self, timestep):
+        '''Fuction to return python struct of group topologi for given timestep'''
+        gt_obj = self._group_tree(timestep)
+        gt={}
+        for n in range (len(self.groups)):
+            group=self.groups[n].name
+            if gt_obj._parent(group)=='': parent=''
+            else: parent = gt_obj._parent(group)
```

Comment:
Exactly one space on each side of `==` and `=`, and always linebreak after `:`.

E.g. `parent = '' if not gt_obj._parent(group) else gt_obj._parent(group)`.